recipes/pythreejs/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.2.0" %}
+
+package:
+    name: pythreejs
+    version: {{ version }}
+
+source:
+    fn: pythreejs-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/p/pythreejs/pythreejs-{{ version }}.tar.gz
+    md5: 7ccfe87baa0fa882a0965d0abd4d0de5
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - jupyter
+    run:
+        - python
+        - ipywidgets >=5.1.3
+
+test:
+    imports:
+        - pythreejs
+
+about:
+    home: https://github.com/jovyan/pythreejs
+    license: BSD
+    summary: A Python / ThreeJS bridge utilizing the Jupyter widget infrastructure.
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - jasongrout
```

Comment:
Again guessing you guys have talked about this. Just doing my due diligence.

Are you ok with being a maintainer here, @jasongrout?